### ISSUE TITLE
Check on fade duration metada in crossfade, implement old behavior by default

### ISSUE THREAD

**Issue by toots** [savonet/liquidsoap#888]
cc @RM-FM

**Comment by toots** [comment]
@RM-FM: you might want to checkout the `default-to-old-crossfade` branch now. I've used this script and it seems to be behaving as expected:
```
def next() =
  path = list.hd(default="",get_process_lines("find /tmp/bla | grep mp3 | sort -R | head -n 1"))

  cross_duration = random.float(min=5.,max=7.)
  fade_in = random.float(min=2.,max=5.)
  fade_out = random.float(min=2.,max=5.)

  cue_in = random.float(min=2.,max=15.)
  cue_out = random.float(min=120.,max=250.)

  req = "annotate:liq_cue_in=\"#{cue_in}\",liq_cue_out=\"#{cue_out}\",liq_cross_duration=\"#{cross_duration}\",liq_fade_in=\"#{fade_in}\",liq_fade_out=\"#{fade_out}\":#{path}"
  print(req)
  request.create(req)
end

s = request.dynamic(length=60.,next)

s = cue_cut(s)

s = crossfade(s)
```

**Comment by RM-FM** [comment]
@toots The `crossfade` seems to ignore meta data - at least for `liq_fade_in`. There is always a fade at the beginning even if we have set it to 0.1 only. Furthermore `cue_cut` still creates ugly endings in many cases (some kind of a "click" at the end of the track).

**Comment by toots** [comment]
liq_fade_in` is the duration of the fade in -- what are you expecting it to do? Also, what source do you have under the `cue_cut`?

It would help to see some logs/script if you could share them.. 🙂

**Comment by RM-FM** [comment]
What I meant is, that we set 0.1 of fade in by meta data but the fade is much longer. It most likely applies the default value. I just took note that the „click“ at the end does occure as well if I‘m using crossfade only (but not cue_cut).

The latter also happens with Master Branch.

**Comment by RM-FM** [comment]
We did some more tests. Obviously `cue_cut` and `crossfade` on the same source are causing very inconstant results. Just now they are not working together at all. As soon as `cue_cut` is set, no more crossfades are executed. 

We just switched back to master branch using the wrapper/function you've provided on Slack as this worked fine so far. The only thing where I'm not sure with it is, whether it considers the fade meta tags or not...?
